fe/fe-core/src/main/java/com/starrocks/system/BackendHbResponse.java

```diff
@@ -63,6 +64,9 @@ public class BackendHbResponse extends HeartbeatResponse implements Writable {
     private int cpuCores;
     @SerializedName(value = "rebootTime")
     private long rebootTime = -1L;
+
+    @SerializedName(value = "statusCode")
+    private TStatusCode statusCode = TStatusCode.OK;
```

Comment:
what is the content of TStatusCode type in serialization mode?

Reply:
the new field will not be brought to the followers/observers giving current read/write implementation in BackendHbResponse.java. need to think about other ways to sync the status with followers/observers.

Reply:
in heartbeatV2, the heartbeat package is synced to the followers/observers with `HbPackage.toGson`, the statusCode can be correctly sync to the follower.